modules/microshift-adding-containers-to-blueprint.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+[id="adding-{product-title}-container-images_{context}"]
+= Adding the {product-title} container images
+
+You can embed {product-title}'s container images into the {op-system-ostree} images so that they are immediately available to the CRI-O container engine after booting. By embedding, images are not pulled over the network from a container registry. In Image Builder, a container image is embedded by adding a reference to it to the Image Builder blueprint.
+
+.Example adding a container image to a blueprint:
+[source,terminal]
+----
+[[containers]]
+source = "<image_pullspec_with_tag_or_digest>"
+----
+
+You need to have the exact list of container image references used by the {product-title} version to embed {product-title} 's container images. Use the `microshift-release-info` RPM package matching the version of the `microshift` RPM in your blueprint.
+
+.Procedure
+
+. Download the `microshift-release-info` RPM package matching your {product-title} version by running the following commands:
++
+[source,terminal]
+----
+$ VERSION=$(rpm -q --queryformat='%{VERSION}-%{RELEASE}.%{ARCH}' microshift)
+----
++
+[source,terminal]
+----
+$ yum download microshift-release-info-${VERSION}
+----
+
+. Extract the release info by running the following command:
++
+[source,terminal]
+----
+$ rpm2cpio microshift-release-info-${VERSION}.rpm | cpio -idmv
+./usr/share/microshift/release/release-aarch64.json
+./usr/share/microshift/release/release-x86_64.json
+----
+
+. Generate the lines to append to your blueprint using the release info for your CPU architecture by running the following command:
++
+[source,terminal]
+----
+$ jq -r '.images | .[] | ("[[containers]]\nsource = \"" + . + "\"\n")' ./usr/share/microshift/release/release-$(uname -i).json
+----
+.Sample output
++
+[source, terminal]
+----
+[[containers]]
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:9945c3f5475a37e145160d2fe6bb21948f1024a856827bc9e7d5bc882f44a750"
+
+[[containers]]
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:82cfef91557f9a70cff5a90accba45841a37524e9b93f98a97b20f6b2b69e5db"
+
+...
```

Comment:
Since this isn't a YAML file, I don't think you need the ellipsis.

Reply:
There are more than 2 images, so the output will be longer than this example. The ellipsis is trying to represent that, it's not part of the literal output.